truss/templates/server/requirements.txt

```diff
@@ -3,7 +3,7 @@
 argparse==1.4.0
 aiocontextvars==0.2.2
 cython==0.29.23
-kfserving==0.5.0
+kserve==0.10.0rc1
```

Comment:
Depending on an rc version is not ideal. May be put a code comment here explaining this exception and any future planss around getting rid of the rc version and any timelines.

Reply:
Good point. Context would be lost there. Will comment.

Reply:
Of note, kserve 10.0 just released a few days ago. Should fast follow to upgrade to the non rc version, but there were some breaking changes I want to wait until we have a base upgraded version out for.